routes/shared.go

```diff
@@ -342,9 +349,92 @@ func (fes *APIServer) putUserMetadataInGlobalState(
 
 	return nil
 }
+func etherToWei(val *big.Int) *big.Int {
+	return new(big.Int).Mul(val, big.NewInt(params.Ether))
+}
+func (fes *APIServer) SendDesoMetamask(recipientPkBytes string) (txnHash *lib.BlockHash, _err error) {
```

Comment:
you also need to add an endpoint to server.go pointing to this. Trace through backend endpoints to see what's the pattern.